### ISSUE TITLE
Valset snapshot logs were not moved to DEBUG

### ISSUE THREAD

**Issue by taariq** [palomachain/paloma#655]
# What is happening?
<details>
  <summary>Section description</summary>
<i>Provide as much context as you can. Give as much context as you can to make it easier for the developers to figure what is happening.</i>
</details>

@webelf101 you have snapshot logs still in INFO and not moved to debug. 

```
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF Added peer module=p2p peer={"Data":{},"Logger":{}}
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF snapshot store module=x/valset store-key-name=valset store-key-string="KVStoreKey{0xc001108890, valset}"
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF get current snapshot last-id=7902 module=x/valset snapshot-validator-size=12
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF snapshot store module=x/valset store-key-name=valset store-key-string="KVStoreKey{0xc001108890, valset}"
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF get current snapshot last-id=7902 module=x/valset snapshot-validator-size=12
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF snapshot store module=x/valset store-key-name=valset store-key-string="KVStoreKey{0xc001108890, valset}"
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF get current snapshot last-id=7902 module=x/valset snapshot-validator-size=12
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF snapshot store module=x/valset store-key-name=valset store-key-string="KVStoreKey{0xc001108890, valset}"
Jan 16 07:05:51 paloma palomad[1187746]: 7:05AM INF get current snapshot last-id=7902 module=x/valset snapshot-validator-size=12
```
More logs here: https://gist.github.com/alipostaci2001/bce07ba81b87eb8da254c0430b6f5d70

# Paloma and pigeon versions and logs
<details>
  <summary>Section description</summary>
<i>Write down paloma version.</i>
<i>Write down pigeon version.</i>
<i>Copy and paste pigeon config file as well as relevant ENV variables.</i>
</details>

Paloma and Pigeon v0.11.5

# How to reproduce?
<details>
  <summary>Section description</summary>
  <i>Please write detailed steps of what you were doing for this bug to appear.</i>
</details>

All validators are overwhelmed with the logs.

# What is the expected behaviour?
<details>
  <summary>Section description</summary>
<i>If you know, please write down what is the expected behaviour. If you don't know, that's ok. We can have a discussion in comments.</i>
</details>

snapshot Logs on DEBUG only.

**Comment by taariq** [comment]
@webelf101 I think we will just need to remove these logs for now. Network halted.

**Comment by taariq** [comment]
Fixed with #656
